org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/ReplModule.java

```diff
@@ -62,7 +62,8 @@
     protected void bindCommands(MapBinder<String, IReplCommand> commandBinder) {
         commandBinder.addBinding("help").to(HelpCommand.class);
         commandBinder.addBinding("load").to(LanguageCommand.class);
-
+        bind(IReplCommand.class).annotatedWith(Names.named("default_command"))
+        .to(HelpCommand.class);
```

Comment:
Instead of binding HelpCommand you could create a dummy implementation in a lambda with a more accurate error message?